PlayList.java

```diff
@@ -108,10 +155,18 @@ public void add(PlayList other) {
      *  If start is negative or greater than size - 1, returns -1.
      */
     private int minIndex(int start) {
-        //// replace the following statement with your code
-        return 0;
+        if (start >= 0 && start < this.size){
+            int index = start , min = this.tracks[start].getDuration();
```

Comment:
To make your code more readable, do not define many variables values within same line.
```suggestion
            int index = start;
            int min = this.tracks[start].getDuration();
```